sdk/core/core-amqp/src/retry.ts

```diff
@@ -118,7 +139,10 @@ async function checkNetworkConnection(host: string): Promise<boolean> {
  * The retries when made are done so linearly on the given operation for a specified number of times,
  * with a specified delay in between each retry.
  *
- * @param {RetryConfig<T>} config Parameters to configure retry operation.
+ * If `exponentialRetry` option is set, then the delay between retries is adjusted to increase
+ * exponentially with each attempt using back-off factor of power 2.
+ *
+ * @param {RetryConfig<T>} config Parameters that define what type of retry will be performed
```

Comment:
The parameters define more than just the type of retry. The previous statement was good enough i.e `Parameters to configure retry operation`